src/store/traction/pacbio/runCreate/run.js

```diff
@@ -238,17 +141,65 @@ const createRunType = ({ id }) => {
   return isNaN(id) ? newRunType : existingRunType
 }
 
+/**
+ * @param {id} - An Integer for the id of the run
+ * @param {run} - A pacbio sequencing run object minus id
+ * @param {plates} - An object of plates
+ * @param {wells} - An object of wells
+ * @param {smrtLinkVersion} - The SMRT Link Version of the run
+ * @returns {Object} - A payload for the run
+ * creates a JSONAPI payload for a run
+ */
+const createPayload = ({ id, run, plates, wells, smrtLinkVersion }) => {
+  return {
+    data: {
+      type: 'runs',
+      id,
+      attributes: {
+        ...run,
+        pacbio_smrt_link_version_id: smrtLinkVersion.id,
+        plates_attributes: Object.values(plates).map(({ plate_number, ...plate }) => {
+          return {
+            plate_number,
+            ...plate,
+            wells_attributes: createWellsPayload(wells[plate_number]),
+          }
+        }),
+      },
+    },
+  }
+}
+
+/**
+ * @param {wells} - An object of wells
+ * @returns {Object} - A payload for the wells
+ */
+const createWellsPayload = (wells) => {
+  // isolate the _destroy attribute from the rest of the wells
+  const { _destroy, ...rest } = wells
+
+  // return the wells with the pools replaced by pool_ids attribute
+  return (
+    Object.values(rest)
+      .map(({ pools: pool_ids, ...attributes }) => {
```

Comment:
I did think about that but it would involve changing it in a lot more places because pools are used in the components. I thought this was a simpler solution for now as it only involved a 1 line change.